src/tools/prted/prted.c

```diff
@@ -811,7 +811,9 @@ int main(int argc, char *argv[])
     prte_finalize();
 
     jdata = prte_get_job_data_object(PRTE_PROC_MY_NAME->nspace);
-    PMIX_RELEASE(jdata);
```

Comment:
You should just remove these lines - the entire job array is released in `prte_finalize`, so you should always get `NULL` returned here.